core/src/main/scala/tamer/config.scala

```diff
@@ -1,39 +1,53 @@
 package tamer
 
+import java.time.{Duration => JDuration}
+
 import zio._
 
-final case class RegistryConfig(url: String, cacheSize: Int)
+final case class RegistryConfig(url: String, cacheSize: Int, expiration: JDuration)
 object RegistryConfig {
   def apply(url: String): RegistryConfig = RegistryConfig(
     url = url,
-    cacheSize = 1000
+    cacheSize = 4,
+    expiration = 1.hour
```

Comment:
more sensible cache size for our use case and an expiration that is configurable